guides-and-tutorials/running-a-signer/signer-quickstart.md

```diff
@@ -0,0 +1,548 @@
+Network: testnet
+Chain ID: 0x80000000
+Database path: ~/stacks-signer/data/signer.sqlite
+Metrics endpoint: 127.0.0.1:9154
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Start the signer
+
+If the outputs of the previous commands are correct, you can proceed and start the signer:
+
+```bash
+~/stacks-signer/stacks-signer run -c ~/stacks-signer/signer-config.toml
+```
+
+## Step 3 - Set Up Your Stacks Node
+
+### Download the stacks-node binary
+
+Official binaries are available from the [Stacks Core releases page on Github](https://github.com/stacks-network/stacks-core/releases). Each release includes pre-built binaries. Download the [latest node release ZIP file](https://github.com/stacks-network/stacks-core/releases/latest) for your server’s architecture and decompress it. Inside of that folder is a `stacks-node` binary. Currently, the latest mainnet and testnet versions that you have to use differ.
+
+Assuming a `Linux x64` machine, the commands to download and uncompress the node binary look like this:
+
+{% tabs %}
+
+{% tab title="Mainnet" %}
+
+```bash
+# Enter the node directory
+cd ~/stacks-node
+
+# Download the node binary zip
+wget https://github.com/stacks-network/stacks-core/releases/download/2.5.0.0.7/linux-glibc-x64.zip
+
+# Unzip the node binary archive
+unzip linux-glibc-x64.zip
+
+# Remove unwanted files
+find ~/stacks-node -type f ! -name 'stacks-node' -delete
+```
+
+{% endtab %}
+
+{% tab title="Testnet" %}
+
+```bash
+# Enter the node directory
+cd ~/stacks-node
+
+# Download the node binary zip
+wget https://github.com/stacks-network/stacks-core/releases/download/3.0.0.0.0-rc5/linux-glibc-x64.zip
+
+# Unzip the node binary archive
+unzip linux-glibc-x64.zip
+
+# Remove unwanted files
+find ~/stacks-node -type f ! -name 'stacks-node' -delete
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+### Create the configuration file
+
+Create the configuration file required to start the node (be sure to replace `<your_token>` with your auth token value):
+
+{% tabs %}
+
+{% tab title="Mainnet" %}
+
+```bash
+# Set environment variables
+AUTH_TOKEN=<your_token> # Used for signer-node authentication, same token as the one set up in the signer configuration
+
+# Create the node's configuration file
+cat <<EOF> ~/stacks-node/node-config.toml
+[node]
+working_dir = "$HOME/stacks-node/data"
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
+bootstrap_node = "02196f005965cebe6ddc3901b7b1cc1aa7a88f305bb8c5893456b8f9a605923893@seed.mainnet.hiro.so:20444,02539449ad94e6e6392d8c1deb2b4e61f80ae2a18964349bc14336d8b903c46a8c@cet.stacksnodes.org:20444,02ececc8ce79b8adf813f13a0255f8ae58d4357309ba0cedd523d9f1a306fcfb79@sgt.stacksnodes.org:20444,0303144ba518fe7a0fb56a8a7d488f950307a4330f146e1e1458fc63fb33defe96@est.stacksnodes.org:20444"
+stacker = true
+
+[burnchain]
+chain = "bitcoin"
+mode = "mainnet"
+peer_host = "bitcoind.stacks.co"
+username = "blockstack"
+password = "blockstacksystem"
+rpc_port = 8332
+peer_port = 8333
```

Comment:
I have a node configured this way and it seems to fall behind quite often (please reach out and we can look at the data).

If that's the case: shouldn't we suggest users to run their own Bitcoin nodes?

Reply:
I could add some comments related to that, but I don't want to complicate this page too much - it's meant to be as simple as possible in order to get users to run a signer without too much hassle. Perhaps we could point them at one of your btc nodes @wileyj?

Reply:
That would be an ideal solution, we just need to make sure the default setup is robust.

Reply:
@aldur @BowTiedDevOps that burnchain section is incredibly outdated. 
if anything, i would use the stacks.org bitcoin node, or better add a section to use a local bitcoin node. 

@aldur to your issue specifically - i've only seen that when geographic latency and endpoint CPU/num conns is an issue. 
ex: for whatever reason, hosts in CET have problems staying connected to btc hosts we have in EST